docs/v3/stores.rst

```diff
@@ -11,3 +14,7 @@ Under construction.
    :caption: Contents:
 
    stores/*/*
+
+.. note::
+   Stores are *not* extension points since they define the mechanism
+   for loading metadata documents such that extensions can be loaded.
```

Comment:
If "extensibility" is defined as a property of a field in an a metadata document, then we don't need this note, because stores are not defined in metadata.